Readme.md

```diff
@@ -6,6 +6,8 @@ FreeSpace2 *S*ource *C*ode *P*roject
 
 Building
 --
+Before you do anything, make sure you have updated your git submodules, either by running git submodule update --init --recursive or by cloning the repository with the --recursive flag.<br/>
```

Comment:
Commands should be put into `code blocks` to separate them from the rest of the text. You can do that by enclosing it in "`" characters.